CONTRIBUTING.md

```diff
@@ -154,6 +154,10 @@ When declaring a record in XML,
   action/menu/views, the convention may not be applicable.
 * Use naming convention defined at the next point
 * The tag `<data>` is only used to set not-updatable data with `noupdate=1`
+* Avoid duplicate `<record id="no_duplicate_id"`. In case of multiple inherit,
```

Comment:
Isn't this like saying "don't use the same variable name for something else"? It's a programming error, not a style directive.